README.md

```diff
@@ -1,7 +1,7 @@
 [![Hibernate team logo](http://static.jboss.org/hibernate/images/hibernate_logo_whitebkg_200px.png)](https://hibernate.org/reactive)
 
 [![Main branch build status](https://github.com/hibernate/hibernate-reactive/workflows/Hibernate%20Reactive%20CI/badge.svg?style=flat)](https://github.com/hibernate/hibernate-reactive/actions?query=workflow%3A%22Hibernate+Reactive+CI%22)
-[![LGPL-2.1 license](https://img.shields.io/badge/License-LGPL%202.1-green.svg?logo=GNU&style=flat)](https://opensource.org/licenses/LGPL-2.1?)
+[![LGPL-2.1 license](https://img.shields.io/badge/License-APACHE%202.0-green.svg?logo=APACHE&style=flat)](https://www.apache.org/licenses/LICENSE-2.0.txt)
```

Comment:
very minor, but would be nice to prefer the opensource.org website:
 - https://opensource.org/licenses/Apache-2.0

Reply:
and the "LGPL-2.1 license"

Reply:
OK